src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -261,11 +262,11 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
       <ProfileInfoListItem title={'Graduation Year:'} contentText={profile.PreferredClassYear} />
     ) : null;
 
-  const cliftonStrengths =
-    profile.CliftonStrengths && (myProf || !profile.CliftonStrengths.Private) ? (
-      <ProfileInfoListItem
-        title="Clifton Strengths:"
-        contentText={
+  const cliftonStrengths = (
+    <ProfileInfoListItem
+      title="Clifton Strengths:"
+      contentText={
+        profile.CliftonStrengths && (myProf || !profile.CliftonStrengths.Private) ? (
```

Comment:
If you use the logic above, then you can also simplify this line to only check if they have clifton strengths:
```suggestion
        profile.CliftonStrengths ? (
```